subsys/dfu/Kconfig

```diff
@@ -11,6 +11,9 @@ menuconfig DFU_TARGET
 
 if (DFU_TARGET)
 
+config DFU_TARGET_ALWAYS_RESTART
+	bool "Always start writing from the start"
```

Comment:
Some help text describing the alternative would be good. Explaining that otherwise, the dfu target will attempt to continue the download where it left off.

Reply:
Added some help text but it's maybe not as comperhensive as it should have been.